src/reconstruction/fastGapFill/AuxillaryFiles/createXMatrix2.m

```diff
@@ -224,5 +224,6 @@
     end
     showprogress(i/length(compounds));
 end
-
-ExchangeRxnMatrix.mets = ExchangeRxnMatrix.mets';
+%Currently we are likely to have a largely blown up S matrix.
+ExchangeRxnMatrix.S = ExchangeRxnMatrix.S(1:numel(ExchangeRxnMatrix.mets),1:numel(ExchangeRxnMatrix.rxns));

```

Comment:
I have no idea. 
its just initialized using spalloc amd I assume, its initialized in a way that allows the code to run in any instance. I was just seeing that there was a size discrepancy which leads to problematic code, and inconsistent models being passed into functions (that admittedly originally didn't care about these things, as they only relied on mets and rxns as size indications....)